client/app/account/billing/history/billing-history.controller.js

```diff
@@ -151,6 +151,18 @@ angular.module("Billing.controllers").controller("Billing.controllers.History",
         $scope.setAction("payDebt", debt, "history");
     };
 
+    this.openValidationModal = function () {
+        const data = {
+            choice: this.invoicesByPostalMail,
+            saveData: this.tmpInvoicesChoice
+        };
+        $scope.setAction("validateInvoicesChange", data);
+    };
+
+    $scope.cancelChoiceModal = function () {
+        self.invoicesByPostalMail = angular.copy(self.tmpInvoicesChoice);
```

Comment:
Do you guys use `angular` instead of `lodash` ? What's the official UX stance on this ? (@jleveugle, don't know who to tag for this anymore)